packages/styles/src/components/header/index.scss

```diff
@@ -64,4 +64,9 @@ post-header {
       background: var(--post-core-color-sandgrey-100);
     }
   }
+  button {
+    @include utilities.high-contrast-mode() {
+      color: ButtonText !important;
+    }
+  }
```

Comment:
This does not seem necessary and overrides the color on hover so it can probably be removed: 
```suggestion
```